ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2

```diff
@@ -1,23 +1,33 @@
 {# eos - router isis #}
-{% if router_isis is defined and router_isis is not none %}
+{% if router_isis is arista.avd.defined %}
```

Comment:
change this to `router_isis.instance` and skip the next one

Reply:
Fix with commit 965ff23